python/sdist/amici/jax/petab.py

```diff
@@ -176,26 +202,53 @@ def _get_measurements(
                 by=petab.TIME
             )
 
-            ts = m[petab.TIME].values
+            ts = m[petab.TIME]
             ts_preeq = ts[np.isfinite(ts) & (ts == 0)]
             ts_dyn = ts[np.isfinite(ts) & (ts > 0)]
             ts_posteq = ts[np.logical_not(np.isfinite(ts))]
+            index = pd.concat([ts_preeq, ts_dyn, ts_posteq]).index
+            ts_preeq = ts_preeq.values
+            ts_dyn = ts_dyn.values
+            ts_posteq = ts_posteq.values
             my = m[petab.MEASUREMENT].values
             iys = np.array(
                 [
                     self.model.observable_ids.index(oid)
                     for oid in m[petab.OBSERVABLE_ID].values
                 ]
             )
+            if (
+                petab.OBSERVABLE_TRANSFORMATION
+                in self._petab_problem.observable_df
+            ):
+                trafo_map = {
+                    petab.LIN: 0,
+                    petab.LOG: 1,
+                    petab.LOG10: 2,
```

Comment:
It would be great to have some enum/constants here, or at least have a reference to where the values are defined/used.